specs/cli/manifest-add.md

```diff
@@ -0,0 +1,180 @@
+    "GitHub": {
+      "x-ms-kiotaHash": "9EDF8506CB74FE44...",
+      "apiDescriptionUrl": "https://raw.githubusercontent.com/github/rest-api-description/main/descriptions/api.github.com/api.github.com.json",
+      "apiDeploymentBaseUrl": "https://api.github.com/",
+      "apiDescriptionVersion": "v1.0",
+      "requests": [
+        {
+          "method": "GET",
+          "uriTemplate": "/repos/{owner}/{repo}"
+        },
+        {
+          "method": "PATCH",
+          "uriTemplate": "/repos/{owner}/{repo}"
+        },,
+        {
+          "method": "DELETE",
+          "uriTemplate": "/repos/{owner}/{repo}"
+        },
+        {
+          "method": "GET",
+          "uriTemplate": "/repos/{owner}/{repo}/actions/artifacts"
+        },
+        {
+          "method": "GET",
+          "uriTemplate": "/repos/{owner}/{repo}/actions/artifacts/{artifact_id}"
+        },
+        {
+          "method": "DELETE",
+          "uriTemplate": "/repos/{owner}/{repo}/actions/artifacts/{artifact_id}"
+        },
+        {
+          "method": "GET",
+          "uriTemplate": "/repos/{owner}/{repo}/actions/cache/usage"
+        }
+      ]
+    }
+  }
+}
+```
+
+_The resulting `apimanifest.json` file (concatenated surface of all APIs) will look like this:_
+
+```jsonc
+{
+  "apiDependencies": {
+    "GraphPlugin": {
+      "x-ms-kiotaHash": "1GFCD345RF3DD98...",
+      "apiDescriptionUrl": "https://aka.ms/graph/v1.0/openapi.yaml",
+      "apiDeploymentBaseUrl": "https://graph.microsoft.com",
+      "apiDescriptionVersion": "v1.0",
+      "requests": [ ... ]
+    },
+    "GitHub": {
+      "x-ms-kiotaHash": "9EDF8506CB74FE44...",
+      "apiDescriptionUrl": "https://raw.githubusercontent.com/github/rest-api-description/main/descriptions/api.github.com/api.github.com.json",
+      "apiDeploymentBaseUrl": "https://api.github.com/",
+      "apiDescriptionVersion": "v1.0",
+      "requests": [
+        {
+          "method": "GET",
+          "uriTemplate": "/repos/{owner}/{repo}"
+        },
+        {
+          "method": "PATCH",
+          "uriTemplate": "/repos/{owner}/{repo}"
+        },,
+        {
+          "method": "DELETE",
+          "uriTemplate": "/repos/{owner}/{repo}"
+        },
+        {
+          "method": "GET",
+          "uriTemplate": "/repos/{owner}/{repo}/actions/artifacts"
+        },
+        {
+          "method": "GET",
+          "uriTemplate": "/repos/{owner}/{repo}/actions/artifacts/{artifact_id}"
+        },
+        {
+          "method": "DELETE",
+          "uriTemplate": "/repos/{owner}/{repo}/actions/artifacts/{artifact_id}"
+        },
+        {
+          "method": "GET",
+          "uriTemplate": "/repos/{owner}/{repo}/actions/cache/usage"
+        }
+      ]
+    }
+  }
+}
+```
+
+## File structure
+```bash
+ └─.kiota
+    └─plugins
+       └─GraphPlugin.yaml
+       └─GitHub.json
+ └─generated
+    └─plugins
+       └─graphplugin
+          └─graphplugin-apimanifest.json
+       └─github
```

Comment:
This seems to be missing the stripped down version of the OpenAPI description that supports the endpoint selection.